src/__pytest__/conftest.py

```diff
@@ -65,162 +65,192 @@ def fake_mapdata_edge_1():
             "edge_basenode": 1,
             "distance": 100,
             "edge_name": "Eksempel Gade",
-            "highway": True,
+            "highway": "highway",
             "maxspeed": 130,
-            "edge_adj": 0,
+            "edge_adj": 2,
         },
         {
             "edge_id": 2,
             "edge_basenode": 1,
             "distance": 200,
             "edge_name": "Eksempel Gade",
-            "highway": False,
+            "highway": "residential",
             "maxspeed": 80,
-            "edge_adj": 2,
+            "edge_adj": 3,
         },
         {
             "edge_id": 3,
             "edge_basenode": 2,
-            "distance": 200,
+            "distance": 300,
             "edge_name": "Eksempel Gade",
-            "highway": False,
+            "highway": "residential",
             "maxspeed": 80,
-            "edge_adj": 1,
+            "edge_adj": 4,
         },
         {
             "edge_id": 4,
             "edge_basenode": 2,
-            "distance": 300,
+            "distance": 400,
             "edge_name": "Eksempel Gade",
-            "highway": False,
+            "highway": "city",
             "maxspeed": 50,
-            "edge_adj": 3,
+            "edge_adj": 5,
         },
         {
             "edge_id": 5,
             "edge_basenode": 3,
-            "distance": 300,
+            "distance": 500,
             "edge_name": "Eksempel Gade",
-            "highway": False,
+            "highway": "city",
             "maxspeed": 50,
-            "edge_adj": 2,
+            "edge_adj": 6,
         },
+        {
+            "edge_id": 6,
```

Comment:
Hvorfor er der behov for en 7. edge?
Tænker det vil være bedre at lave en ny "fixture collection" som modellere et mere sammenhængende netværk (som ser ud til at være det du ønsker?) Men at vi stadig skal have et "bare bones" / simpelt "straight line" netværk?